ui/src/actions/queue.js

```diff
@@ -1,7 +1,7 @@
 import { showErrorPanel } from './general';
-import { mountSample } from './sampleChanger'; // eslint-disable-line import/no-cycle
```

Comment:
`no-cycle` is now disabled. It slows down linting _significantly_ because it has to perform module resolution. Vite can detect when import cycles cause issues, so it's not worth detecting this during linting.